build.gradle

```diff
@@ -55,21 +55,37 @@ ext {
             junit5JupiterApi    : [group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.8.2'],
             junit5JupiterEngine : [group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.8.2'],
             junit5VintageEngine : [group: 'org.junit.vintage', name: 'junit-vintage-engine', version: '5.8.2'],
+            junitJupiterParams  : [group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: '5.8.2'],
             junitPlatform       : [group: 'org.junit.platform', name: 'junit-platform-runner', version: '1.8.2'],
             junitPlatformCommons: [group: 'org.junit.platform', name: 'junit-platform-commons', version: '1.8.2'],
             junitPlatformEngine : [group: 'org.junit.platform', name: 'junit-platform-engine', version: '1.8.2'],
+            junitPlatformLauncher : [group: 'org.junit.platform', name: 'junit-platform-launcher', version: '1.8.2'],
+            junitPioneer        : [group: 'org.junit-pioneer', name: 'junit-pioneer', version: '1.6.1'],
             hamcrest            : [group: 'org.hamcrest', name: 'hamcrest-core', version: '1.3'],
             junit_dataprovider  : [group: 'com.tngtech.java', name: 'junit-dataprovider', version: '1.11.0'],
             mockito             : [group: 'org.mockito', name: 'mockito-core', version: '4.4.0'],
             assertj             : [group: 'org.assertj', name: 'assertj-core', version: '3.22.0'],
             assertj_guava       : [group: 'org.assertj', name: 'assertj-guava', version: '3.4.0'],
 
+            findBugsAnnotations : [group: 'com.google.code.findbugs', name: 'annotations', version: '3.0.1'], // SpotBugs version 4 requires Java 9+
+
+            // Dependencies for tool integrations
+            surefireApi         : [group: 'org.apache.maven.surefire', name: 'surefire-api', version: '3.0.0-M7'],
+            gradleApi           : [group: 'dev.gradleplugins', name: 'gradle-api', version: '7.4.1'],
```

Comment:
`gradleApi` and `byteBuddy` are not yet used anywhere (they will be used for Gradle filtering)